models/yang/sonic/sonic-mgmt-vrf.yang

```diff
@@ -0,0 +1,45 @@
+module sonic-mgmt-vrf {
+    namespace "http://github.com/Azure/sonic-mgmt-vrf";
+    prefix mgmt-vrf;
+
+    import sonic-common {
+        prefix scommon;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC MGMT VRF";
+
+    revision 2019-10-08 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-mgmt-vrf {
+        container MGMT_VRF_CONFIG {
+            description "Management vrf configuration.";            
+
+            list MGMT_VRF_CONFIG_LIST {
+                key "mgmt-vrf-name";
+
+                leaf mgmt-vrf-name {
```

Comment:
please avoid using mgmt in field names since it is within MGMT_VRF_CONFIG container.

Reply:
will move to vrf-global-name